.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: Create a release on tag
+on: 
+  push: 
+    tags:
+      - "v*.*.*"
+jobs:
+  build: 
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v4
+        with: 
+          fetch-depth: 0
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version-file: 'package.json'
+          cache: 'npm'
+
+      - name: Install Node dependencies
+        run: npm ci
+
+      - name: Build the package
+        run: |
+            npm run build:kinto-single
+            cd build
+            tar -cvf "$RUNNER_TEMP/kinto-admin-release.tar" *
+
+      - name: Create release
+        uses: softprops/action-gh-release@v1
+        with:
+          draft: true
+          fail_on_unmatched_files: true
+          files: ${{ runner.temp }}/kinto-admin-release.tar
```

Comment:
This seems like it works just fine, so consider this comment optional, but it might be nice if we don't have to rely on a 3rd party action for this. It looks like the `gh` CLI has a [release](https://cli.github.com/manual/gh_release_create) command built in -- maybe that'd work?

Reply:
I settled on this one after finding [actions/create-release](https://github.com/actions/create-release) being archived/unsupported and it's read-me mentioning this one and it seeming  simple enough for our needs.

I'll look into the cli option to see what that takes.

Reply:
Implemented!